cleaner.go

```diff
@@ -0,0 +1,362 @@
+// Copyright 2017 Canonical Ltd.
+// Licensed under the LGPLv3, see LICENCE file for details.
+
+package txn
+
+import (
+	"fmt"
+	"gopkg.in/mgo.v2"
+	"gopkg.in/mgo.v2/bson"
+	"os"
+	"strconv"
```

Comment:
Sorry about the grouping here and the comment lengths. I switched to Gogland to try it out, and it doesn't do auto-text width (that I'm aware of) like vim did, nor does it supports vims rewrap keys.
The imports are because it has decided that all imports should be in one section, and it tries to be "helpful" and fix them for you. I have to keep putting them back even though I've tried to remove all of its "auto imports" features.
Sorry about the noise.